app/views/report_mailer/registrar_import_email.html.erb

```diff
@@ -45,3 +46,14 @@
     <% end %>
   </ul>
 <% end %>
+
+<% if @results[:new_degree_periods].any? %>
+  <p>The following new degree periods can be assigned Archivematica accession numbers:</p>
```

Comment:
Not blocking: I kinda want to lean towards slightly stronger language...

`The following new degree periods must be assigned Archivematica accession numbers or publication will not be possible for associated theses` or something like that. `can` feels like it's optional which it isn't if this is our solution to failing to do this could lead to having to re-save hundreds of theses during processing if not done early.